lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -1904,20 +1902,6 @@ def setUp(self):
         # (comment because pylint C0103(invalid-name))
         # self.maxDiff = None
 
-    def test_beta_tester_must_not_earn_cert(self):
```

Comment:
Testing assumption -- are we removing this test because we already have tests (a stronger test too) internal to the Certificates Django app for this functionality?

Reply:
Yep